ADOPTERS.md

```diff
@@ -5,6 +5,6 @@ This really helps the project to gain momentum and credibility. It's a small con
 
 | Organization | Contact | Description of Use |
 |--------------|---------|--------------------|
+| Sovereign Cloud Stack | https://scs.community/ | Currently in the process of evaluating how to generate a complete SBOM of our stack and secure the supply chain. We plan to both use ORT to analyze our reference implementation as well as to add ORT to our [recommended operator's toolset]([https://scs.community/about/](https://scs.community/about/#technological-vision)).  |
```

Comment:
Something about the link syntax for the "recommended operator's toolset" seems to be wrong.